tests/test_app.py

```diff
@@ -27,27 +40,39 @@ def testPutObject(client):
     assert response.status_code == 201
 
 
-def testPutDuplicateKey(client):
+def test_put_duplicate_key(client):
     f = open("./InputTetris.png", "rb")
     data = dict(file=(BytesIO(f.read()), "OutputTetris.png"))
 
-    response = client.put(
+    first_response = client.put(
+        "/S4/PutObject?Key=TestKey", content_type="multipart/form-data", data=data
+    )
+    assert first_response.status_code == 201
+
+    data = dict(file=(BytesIO(f.read()), "OutputTetris.png"))
```

Comment:
Not sure why, but if I remove this line, there is a "ValueError: I/O operation on closed file" even though data is set above.